packages/cli/commands/charm.ts

```diff
@@ -156,24 +156,31 @@ export const charm = new Command()
     `ct charm link ${EX_ID} ${EX_COMP} bafycharm1/data/users/0/email bafycharm2/config/primaryEmail`,
     `Link deep nested field including array access.`,
   )
+  .example(
+    `ct charm link ${EX_ID} ${EX_COMP} baedreiahv63wxwgaem4hzjkizl4qncfgvca7pj5cvdon7cukumfon3ioye bafycharm1/allCharms`,
+    `Link well-known charms list to charm field.`,
+  )
   .arguments("<source:string> <target:string>")
   .action(async (options, sourceRef, targetRef) => {
     const spaceConfig = parseSpaceOptions(options);
 
-    // Parse source reference (charmId/path/to/field)
-    const sourceParts = sourceRef.split("/");
-    if (sourceParts.length < 2) {
-      throw new ValidationError(
-        `Invalid source reference format. Expected: charmId/path/to/field`,
-        { exitCode: 1 },
-      );
+    // Parse source reference - handle both charmId/path and well-known IDs
+    let sourceCharmId: string;
```

Comment:
If turning this into a `parseLink` function in #1285, could be `parseLink(ref: string, { allowWellKnown?: boolean })` or something